docs/src/references/mappers/c8y-mapper.md

```diff
@@ -913,18 +904,9 @@ te/device/main///twin/c8y_Hardware
 }
 ```
 
-:::warning
-The following keys in the `inventory.json` file are also ignored:
-
-* `name`
-* `type`
-
-as they are included in the entity registration message and can only be updated with another registration message.
-:::
-
 ### Updating entity type in inventory
 
-After updating the inventory with `inventory.json` file contents, 
+If not included in the `inventory.json` file,
 the device `type` of the main device, set using the `device.type` tedge config key,
 is also updated in the inventory with the following message:
```

Comment:
I'm starting to question this section. As the type which is published to Cumulocity can be set via multiple methods (from a users perspective), e.g.

* inventory.json => /twin
* tedge config device.type => /twin
* registration message => /twin

Then the mapper only care about translating the `/twin` values to the Cumulocty json over MQTT topic. So do we need to mention the inventory.json files here?

Reply:
Sure. Currently the `tedge config device.type` value is not published to the twin topic but directly published to the cloud. I'll change that and then just mention here that the value of this setting is also published to the twin topic. And also stress the precedence of `inventory.json` value over this config setting.

Reply:
Resolved by 6acf7ad

For the sake of consistency, decided to publish the `name` of the device as well along with the `type` to their respective `twin/` topics.